common/src/main/java/org/apache/drill/common/exceptions/UserException.java

```diff
@@ -37,6 +37,7 @@
  *
  * @see org.apache.drill.exec.proto.UserBitShared.DrillPBError.ErrorType
  */
+@SuppressWarnings("serial")
```

Comment:
Should there instead be in this class :
     private static final long serialVersionUID = -3796081521525479249L;
as does its super-class DrillRuntimeException, and as does its super-class RuntimeException, and as does its super-class Exception, and as does its super-class Throwable (which implements Serializable ........) ??
(Not sure how to pick the specific UID number ...)